### DISCUSSION TITLE
Import Output Overflow

### DISCUSSION THREAD

**Discussion by threelegchicken** [beetbox/beets#5344] [General]
Hi, beets.readthedocs.io shows that I should expect this type of output when I import:
![image](https://github.com/beetbox/beets/assets/20527165/d34e6e6b-f36d-4490-a429-eee3467366f8)

This is what I get:
![image](https://github.com/beetbox/beets/assets/20527165/ba829ed2-3ee3-49b3-9a48-d4dc22124639)

Is there some way to clean this up? Running on Windows 10 Pro.

**Comment by Serene-Arc** [comment]
What are you using as a terminal? As in the program that gives you the command prompt? Those weird little strings of text and numbers are colour codes; rather than changing the colour of the text like it's supposed to do, it's just printing them.

You can disable colour and it will get rid of that as well, but colour is nice. [This is the option](https://beets.readthedocs.io/en/stable/reference/config.html#id82) for disabling colour.

**Reply by threelegchicken** [reply]
I am using the default Win 10 console. Your suggestion works perfectly. Is there any particular console that works well/better with beets?

**Reply by Serene-Arc** [reply]
Those are ANSI colour codes, which are a standard (no idea why Windows doesn't support them out of the box, they've been around for about 40 years plus), so most will support it. If you're confident, you can enable them in Windows through a registry key change, see [here](https://www.codeproject.com/Tips/5255355/How-to-Put-Color-on-Windows-Console).

**Reply by threelegchicken** [reply]
I got the registry updated and now I can see in color. The dog days are over.

**Reply by Serene-Arc** [reply]
Wonderful :)
